mobile-menu.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-multi-str */
+/* eslint-disable no-plusplus */
+const hdr = document.querySelector('.pg-hdr'); // the header element
+const btn = document.querySelector('.btn'); // the opening button
+const logo = document.querySelector('.pg-hdr h3'); // The logo name
+const navBar = document.querySelector('.dsk-btn'); // The desktop navigation bar
+const navLinks = document.querySelectorAll('.nav-link'); // The navigation links inside the nav bar
+const rightArrow = document.querySelectorAll('.r-arrow'); // The right arrow after the nav links
+const navIcon = document.querySelector('.nav-icon'); // The mail icon
+
+/** ******* The closig button *********** */
```

Comment:
- [ ] OPTIONAL
It is a good practice to leave out any comments in your code hosted on the repository, so as to keep your code cleaner. Let's endeavor to improve upon this aspect